backstack/build.gradle.kts

```diff
@@ -49,3 +50,11 @@ kotlin {
 android { namespace = "com.slack.circuit.backstack" }
 
 androidComponents { beforeVariants { variant -> variant.enableAndroidTest = false } }
+
+baselineProfile {
+  // https://issuetracker.google.com/issues/282127523
+  baselineProfileOutputDir = "../../src/androidMain/generated/baselineProfiles"
+  filter { include("com.slack.circuit.backstack.**") }
```

Comment:
What is this for?

Reply:
This tells it which package(s) to filter in from the source profile